modules/testing/default.nix

```diff
@@ -38,6 +38,52 @@ let
         environment.variables = adminEnv;
       };
     };
+
+  portForwarding =
+    { config, lib, ... }:
+    with lib;
+    let
+      cfg = config.openstack-testing;
+    in
+    {
+      options.openstack-testing = {
+        enable = mkEnableOption "Enable port forwarding." // {
+          default = true;
```

Comment:
nit: do we do this elswhere? If not, I'd prefer to have a `enable = true;` somewhere